### DISCUSSION TITLE
Fsrmod does not work on epic Cyberpunk2077

### DISCUSSION THREAD

**Discussion by Tianxingxs** [community/community#80907] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I tested that this mod does not work properly on epic.
![Cyberpunk 2077 Screenshot 2023 12 19 - 21 56 25 80](https://github.com/community/community/assets/154341944/793747e7-e307-4ede-ba36-7428b85cc3cb)

**Comment by queenofcorgis** [comment]
Hi @Tianxingxs ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Fsrmod or Cyberpunk 2077. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by Tianxingxs** [reply]
It's a pity that I didn't know how to use Github when I first came here, and I didn't know how to delete this post
